api-v3-stable.rst

```diff
@@ -0,0 +1,48 @@
+.. post:: Feb 1, 2021
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+API v3 is now stable
+=====================
+
+We are excited to announce that our `API v3 <https://docs.readthedocs.io/page/api/v3.html>`_ has reached a stable release,
+and is now available for all Read the Docs users.
+Since we :doc:`announced the API v3 beta </api-v3-beta>`,
+we have been adding extra functionality and bug-fixing minor issues based on user feedback.
+
+The new API v3 *is not* a fully replacement (yet!) of API v2,
```

Comment:
It feels weird announcing api v3 as stable, but not being a replacement of v2. What are we missing from v2? I think it was just internal usage endpoints.

Reply:
There are some things that are not migrated yet: `/api/v2/search/`, `/api/v2/embed/`, `/api/v2/footer_html/` and others.